jmx/src/main/java/com/lafaspot/jmetrics/monitor/ContainerMonitor.java

```diff
@@ -0,0 +1,650 @@
+/*
+ * Copyright [yyyy] [name of copyright owner]
+ * 
+ * ====================================================================
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *  ====================================================================
+ */
+
+package com.lafaspot.jmetrics.monitor;
+
+import java.lang.management.ManagementFactory;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.NotCompliantMBeanException;
+import javax.management.ObjectName;
+
+import com.lafaspot.jmetrics.annotation.Metric;
+import com.lafaspot.jmetrics.annotation.MetricCheck;
+import com.lafaspot.jmetrics.annotation.MetricClass;
+import com.lafaspot.jmetrics.datatype.TimeValue;
+
+/**
+ * Records incoming http request stats in an interval.
+ *
+ */
+@MetricClass(enable = true, name = "ContainerMonitor", applications = { "admin" })
+public class ContainerMonitor implements ContainerMonitorMBean {
```

Comment:
change the bean and the monitor class to a jmx_sample package

Reply:
Done